challenge.md

```diff
@@ -31,20 +31,20 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
+ <!-- Answer Starts Here --> It is an absolute path. It shows where the file originated.
```

Comment:
Right answer, but the explanation is wrong. The reason that this is an absolute path is that it will always point to the same location, _independent of wherever *we* might be_. Whether we're at `/`, `~`, or `/blah/blah/blah`, the path above will always point to the same place.